aspnetcore/blazor/components.md

```diff
@@ -28,7 +28,7 @@ Components can be authored using the *.cshtml* file extension as long as the fil
 
 The UI for a component is defined using HTML. Dynamic rendering logic (for example, loops, conditionals, expressions) is added using an embedded C# syntax called [Razor](xref:mvc/views/razor). When an app is compiled, the HTML markup and C# rendering logic are converted into a component class. The name of the generated class matches the name of the file.
 
-Members of the component class are defined in an `@functions` block (more than one `@functions` block is permissible). In the `@functions` block, component state (properties, fields) is specified with methods for event handling or for defining other component logic.
+Members of the component class are defined in an `@code` block (more than one `@code` block is permissible). In the `@code` block, component state (properties, fields) is specified with methods for event handling or for defining other component logic.
```

Comment:
@danroth27, is it worth mentioning `@functions` will continue to work?

Reply:
Maybe something like `>Info: In previous versions @functions blocks were used. This will still work, and you might see it in legacy code, but we recommend using the more clearly named @code directive going forward.`?

Reply:
Yup, what @ryanbrandenburg said :smiley: